### PR TITLE
Fix naming for CLI plugin list command

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Fix `mattermost plugin list` to reference plugins as enabled/disabled instead of active/inactive.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-19915

### CODE REVIEW

File: cmd/mattermost/commands/plugin.go [hunk 1]
```diff
@@ -172,12 +172,12 @@ func pluginListCmdF(command *cobra.Command, args []string) error {
...
 	}
 
-	CommandPrettyPrintln("Listing active plugins")
+	CommandPrettyPrintln("Listing enabled plugins")
 	for _, plugin := range pluginsResp.Active {
 		CommandPrettyPrintln(plugin.Manifest.Name + ", Version: " + plugin.Manifest.Version)
 	}
 
-	CommandPrettyPrintln("Listing inactive plugins")
+	CommandPrettyPrintln("Listing disenabled plugins")
```
Comment (marianunez) [inline cmd/mattermost/commands/plugin.go]:
```suggestion
	CommandPrettyPrintln("Listing disabled plugins")
```

Comment (hanzei) [reply cmd/mattermost/commands/plugin.go]: :man_facepalming: thanks for catching this

File: cmd/mattermost/commands/plugin.go [hunk 2]
```diff
@@ -172,12 +172,12 @@ func pluginListCmdF(command *cobra.Command, args []string) error {
 		return errors.New("Unable to list plugins. Error: " + appErr.Error())
 	}
 
-	CommandPrettyPrintln("Listing active plugins")
+	CommandPrettyPrintln("Listing enabled plugins")
 	for _, plugin := range pluginsResp.Active {
```
Comment (marianunez) [inline cmd/mattermost/commands/plugin.go]: To be consistent, we should also update the `PluginResponse` [here]( https://github.com/mattermost/mattermost-server/blob/master/model/plugins_response.go#L15-L18) so the code has the proper naming: 
```suggestion
 	for _, plugin := range pluginsResp.Enabled {
```

File: cmd/mattermost/commands/plugin.go [hunk 3]
```diff
@@ -50,7 +50,7 @@ var PluginDisableCmd = &cobra.Command{
 var PluginListCmd = &cobra.Command{
 	Use:     "list",
 	Short:   "List plugins",
-	Long:    "List all active and inactive plugins installed on your Mattermost server.",
+	Long:    "List all enable and disabled plugins installed on your Mattermost server.",
```
Comment (gabrieljackson) [inline cmd/mattermost/commands/plugin.go]:
```suggestion
	Long:    "List all enabled and disabled plugins installed on your Mattermost server.",
```

File: cmd/mattermost/commands/plugin.go (hunk 2 above)
Comment (hanzei) [reply cmd/mattermost/commands/plugin.go]: Unfortunately that would be a breaking change. I can leave a comment in the code, if you want.

**Comment by hanzei** [comment]
There are a couple of other instances where we use the term `active plugin` but they are not visible to the user. Also, I'm unsure if they can simply be replaced with `enabled/disbaled plugin` so I'm leaving them.

**Comment by marianunez** [comment]
> There are a couple of other instances where we use the term `active plugin` but they are not visible to the user. Also, I'm unsure if they can simply be replaced with `enabled/disbaled plugin` so I'm leaving them.

Agreed that the `active` plugin is used interchangeably with enable plugin in several places in the code so this would be a larger change out of scope for this PR.

**Pull request merged**